pages/core/i2c.rst

```diff
@@ -0,0 +1,20 @@
+I2C
+***
+
+.. highlight:: python
+   :linenothreshold: 4
+
+.. figure:: /images/i2c-overview.png
+   :width: 270
+
+This is a placeholder to describe I2C. In the meantime the following links
+may be useful:
```

Comment:
This isn't particularly helpful to have in Trunk, maybe as a comment in the .rst file, but I think just having the links by themselves would be a nicer experience than a message about WIP that the user can't really take action on

Reply:
Fixed in 9e2e99c